ios/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyAccessedAPITypes</key>
+	<array>
```

Comment:
Should `NSPrivacyAccessedAPICategorySystemBootTime` be added as well ?

Reply:
Yeah need NSPrivacyAccessedAPICategorySystemBootTime aswell

Reply:
Done 🙂